### PR TITLE
DRUP-1118: #ready adds alt property to field_event_thumbnail image field

### PR DESCRIPTION
**Pull request by z3cka**

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_exhibition/uclalib_exhibition.features.field_instance.inc [hunk 1]
```diff
@@ -338,6 +338,7 @@ function uclalib_exhibition_field_default_field_instances() {
         'insert_styles' => array(
           'image' => 0,
           'image_custom_crop' => 0,
+          'image_staff_profile_photo' => 0,
```
Comment (emb03) [inline www/sites/all/modules/features/uclalib_exhibition/uclalib_exhibition.features.field_instance.inc]: @z3cka this was not on prod, creeping back in?

Comment (z3cka) [inline www/sites/all/modules/features/uclalib_exhibition/uclalib_exhibition.features.field_instance.inc]: so weird... this is from the prod db

**Comment by z3cka** [comment]
deploying now

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

**Pull request merged**